test/unit/settings_test.py

```diff
@@ -125,18 +126,24 @@ def test_none_quarters_raise(self: Self):
             _ = EpaCemsSettings(quarters=None)
 
 
-class TestEIA860Settings:
+class TestEia860Settings:
     """Test EIA860 setting validation."""
 
-    def test_860m(self: Self):
-        """Test validation error is raised when eia860m date is within 860 years."""
-        settings_cls = Eia860Settings()
-        original_eia80m_year_month = settings_cls.eia860m_year_months
-        settings_cls.eia860m_year_months = ["2019-11"]
```

Comment:
This assignment is invalid, because the class is immutable.  It was "working" before because this was `ClassVar` but it was actually bad -- the assignment was clobbering the value built into the class.

A simpler / less invasive way to check that we get an error when there's overlap between the EIA-860 and EIA-860M years is to explicitly give the class constructor dates which we know will clash, as is happening below now.